graphs.go

```diff
@@ -0,0 +1,30 @@
+package cwl
+
+// Graphs represents "$graph" field in CWL.
+type Graphs []*Root
+
+// Graph represents an element of "steps"
+type Graph struct {
+	Run *Root
+}
+
+// New constructs "Graphs" from interface.
+func (steps Graphs) New(i interface{}) Graphs {
```

Comment:
If `this` object is not used, it can be `_` like this

```go
func (_ Graphs) New(i interface{}) Graphs {
    ...
}
```

I'm now refactoring to make other constructor methods like this.